src/com/library/Entities/Librarian.java

```diff
@@ -0,0 +1,54 @@
+package com.library.Entities;
+
+import com.library.Enums.Experience;
+
+import java.time.LocalDate;
+
+public class Librarian extends Staff {
+    private Experience experience;
+    private double salaryBonus; //depends on experience
+
+    public Librarian(int id, String firstName, String lastName, String phone, double salary, LocalDate hireDate, Experience experience) {
+        super(id, firstName, lastName, phone, salary, hireDate);
+        this.experience = experience;
+        switch (experience) {
```

Comment:
[Fix] aceasta logica ar trebui sa fie intr-un service, nu aici, si se poate seta salaryBonus de unde se creaza si obiectul. Daca vine o recesiune si trebuie sa schimbam aceasta logica, in loc sa schimbam implementarea din serviciu, aici o sa trebuiasca sa modificam entitatea, care nu ar trebui sa se schimbe decat daca adaugam sau scoatem campuri